app/models/application_qualification.rb

```diff
@@ -1,15 +1,11 @@
 class ApplicationQualification < ApplicationRecord
   belongs_to :application_form
 
+  scope :degrees, -> { where level: 'degree' }
+
   enum level: {
     degree: 'degree',
     gcse: 'gcse',
     other: 'other',
   }
-
-  enum subject: {
-    maths: 'maths',
-    english: 'english',
-    science: 'science',
```

Comment:
Ah, had to remove it in order to proceed. See https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/418/commits/b1f770d8e61a15f391f3bf8ce4db5ff1ce9ecf80.

If this stayed in, then when we try to save the `subject` of a degree, it is not possible to do so as it's not maths, english or science. This instead should be in a `CandidateInterface::GcseForm` form object instead of `ApplicationQualification`.

Reply:
👍 Makes sense, should we move it into that model as part of this?

Reply:
There currently isn't a `CandidateInterface::GcseForm` yet. Should I create one? 🤔 Reckon @alexdesi will have created one though.

Reply:
Ah yes, cool, then ship this!